generic/generic_font-1.2.3.recipe

```diff
@@ -19,21 +19,24 @@ DISABLE_SOURCE_PACKAGE=yes
 PROVIDES="
 	generic_font = $portVersion
 	"
-REQUIRES=""
+REQUIRES="
+	"
 
-BUILD_REQUIRES=""
-BUILD_PREREQUIRES=""
+BUILD_REQUIRES="
+	"
+BUILD_PREREQUIRES="
+	"
 
 BUILD()
 {
-	true
+	: # means nothing.
 }
```

Comment:
I think the BUILD function can be omitted entirely in this case (only INSTALL is required).

Reply:
Not there yet, but thanks for keeping an eye out, time to bring this to current status.